packages/blocks/src/api/raw-handling/paste-handler.js

```diff
@@ -189,6 +190,7 @@ export function pasteHandler( {
 			}
 
 			const filters = [
+				nestedListedConverter,
```

Comment:
@ellatrix in https://github.com/WordPress/gutenberg/pull/46775, you mentioned updating `filterInlineHTML`, but I found that this is not always called. I registered a transformer function here - is this ok?

Reply:
You'll need to add it both here and `filterInlineHTML`.

Reply:
I've added the transform function there too. Still need to test it though.